include/threads/ThreadState.h

```diff
@@ -21,31 +21,34 @@ class Level
     uint32_t id;
 
     // Number of nested OpenMP constructs
-    int depth = 0;
+    int32_t depth = 0;
```

Comment:
Because the raw bytes underlying this object are serialised, sent across the wire and deserialised, I wanted to be explicit about the width of each field.